src/units/unit.cpp

```diff
@@ -1528,10 +1485,10 @@ static bool matches_ability_filter(const config & cfg, const std::string& tag_na
 	if(!int_matches_if_present(filter, cfg, "value"))
 		return false;
```

Comment:
"value" "multiply" and divide" can also be negative, i don't undestand why you don't use matches_if_present_or_negative for them.

Reply:
oh, excuse me, i trust what int_matches_if_present_or_negative was for support negative numberin range.